### PR TITLE
QUICK-FIX Change test of audit contains snapshotable control after deleting original control

### PR DESCRIPTION
**Pull request by AleksNeStu**
This PR unskip and change according to actual expected behavior of existing test: "test_audit_contains_snapshotable_control_after_deleting_original_control".

### CODE REVIEW

File: test/selenium/src/lib/constants/locator.py [hunk 1]
```diff
@@ -828,23 +828,33 @@ class WidgetAdminEvents(object):
...
-                                           PIN_CONTENT + GET_PERMALINK)
-  BUTTON_3BBS_OPEN_UNDER_AUDIT = (By.CSS_SELECTOR, PIN_CONTENT + OPEN)
-  BUTTON_3BBS_DELETE_UNDER_AUDIT = (By.CSS_SELECTOR, PIN_CONTENT + DELETE)
+  BUTTON_3BBS_EDIT_UNDER_AUDIT = (
+      By.CSS_SELECTOR, PIN_CONTENT + INFO_3BBS_DROPDOWN + EDIT)
+  BUTTON_3BBS_GET_PERMALINK_UNDER_AUDIT = (
+      By.CSS_SELECTOR, PIN_CONTENT + INFO_3BBS_DROPDOWN + GET_PERMALINK)
+  BUTTON_3BBS_OPEN_UNDER_AUDIT = (
+      By.CSS_SELECTOR, PIN_CONTENT + INFO_3BBS_DROPDOWN + OPEN)
+  BUTTON_3BBS_OPEN_UNDER_AUDIT_DISABLED = (
```
Comment (SergeyRabtsau) [inline test/selenium/src/lib/constants/locator.py]: There is no need to create such locator

File: test/selenium/src/lib/element/widget_info.py [hunk 2]
```diff
@@ -22,6 +22,9 @@ def __init__(self, driver, is_under_audit):
     self.open_locator = (
         self._locators.BUTTON_3BBS_OPEN_UNDER_AUDIT if is_under_audit
         else self._locators.BUTTON_3BBS_OPEN)
+    self.open_locator_disabled = (
```
Comment (SergeyRabtsau) [inline test/selenium/src/lib/element/widget_info.py]: There is no need to create such specific locator

**Review by SergeyRabtsau — CHANGES_REQUESTED** [review]

File: test/selenium/src/lib/element/widget_info.py [hunk 3]
```diff
@@ -33,12 +36,13 @@ def select_open(self):
     """Select open button in 3BBS dropdown modal."""
     base.Button(self._driver, self.open_locator).click()
 
-  def is_open_exist(self):
-    """Find open button in 3BBS dropdown modal.
-    Return: True if open button is exist,
-            False if open button is not exist.
+  def is_open_enabled(self):
+    """Find open button in 3BBS dropdown modal and check if enabled.
```
Comment (SergeyRabtsau) [inline test/selenium/src/lib/element/widget_info.py]: Re-implement this method by introducing reusable methods in `selenium_utils.py` module
```python
def get_parent(driver, element):
  """Get parent element of curent element using JS."""
  return driver.execute_script("return arguments[0].parentNode;", element)

def is_value_in_parent_attr(dirver, element, attr, value):
  parent = get_parent(dirver, element)
  return is_value_in_attr(parent, attr, value)
```

**Review by SergeyRabtsau — CHANGES_REQUESTED** [review]

**Comment by AleksNeStu** [comment]
@SergeyRabtsau I have changed approach of finding 'disabled' element,  please take a look.

File: test/selenium/src/tests/test_snapshots.py [hunk 4]
```diff
@@ -167,6 +166,8 @@ def test_audit_contains_snapshotable_control_after_deleting_original_control(
                        get_list_objs_from_tree_view(src_obj=audit))
     assert [expected_control] == actual_controls, (
         messages.ERR_MSG_FORMAT.format([expected_control], actual_controls))
+    (webui_service.ControlsService(selenium).
```
Comment (SergeyRabtsau) [inline test/selenium/src/tests/test_snapshots.py]: This test should be separated or parametrized into 2 tests:
1. delete control and do not update to latest version
2. delete control and update to latest version

I both cases asserts should go after required actions.

**Review by SergeyRabtsau — CHANGES_REQUESTED** [review]

File: test/selenium/src/tests/test_snapshots.py (hunk 4 above)
Comment (AleksNeStu) [reply test/selenium/src/tests/test_snapshots.py]: Done.
```Python
test_audit_contains_snapshotable_control_after_deleting_control
test_update_snapshotable_ver_after_deleting_original_control
```

**Pull request merged**